src/renderer/dx/DxRenderer.cpp

```diff
@@ -1031,6 +1037,18 @@ try
 }
 CATCH_LOG()
 
+void DxEngine::SetPixelShaderImagePath(std::wstring_view value) noexcept
+try
+{
+    if (_pixelShaderImagePath != value)
+    {
+        _pixelShaderImagePath = std::wstring{ value };
+        _recreateDeviceRequested = true;
+        LOG_IF_FAILED(InvalidateAll());
+    }
+}
+CATCH_LOG()
+

```

Comment:
You can revert the changes to `src/renderer/dx` as it doesn't use `_pixelShaderImagePath`.

Reply:
I think I need to implement the interface? But it can be implemented as a NOP?

Reply:
You don't need it, since you already provided a no-op implementation in `IRenderEngine`.

Reply:
Ok

Reply:
Will change to NOPs

Reply:
done